dotnet/src/Connectors/Connectors.MistralAI/MistralAIToolCallBehavior.cs

```diff
@@ -160,17 +167,11 @@ internal override void ConfigureRequest(Kernel? kernel, ChatCompletionRequest re
     /// <summary>
     /// Represents a <see cref="MistralAIToolCallBehavior"/> that provides a specified list of functions to the model.
     /// </summary>
-    internal sealed class EnabledFunctions : MistralAIToolCallBehavior
+    internal sealed class AnyFunction(IEnumerable<KernelFunction> functions, bool autoInvoke) : MistralAIToolCallBehavior(autoInvoke)
```

Comment:
Maybe consider adding Choice to those to avoid confusion on the `Auto` I was almost thinking this was auto until I saw the `bool autoInvoke`.

```suggestion
    internal sealed class AnyFunctionChoice(IEnumerable<KernelFunction> functions, bool autoInvoke) : MistralAIToolCallBehavior(autoInvoke)
```